packages/js-utils/src/unique-by.ts

```diff
@@ -0,0 +1,10 @@
+const defaultCompare = ( a: any, b: any ): boolean => a === b;
```

Comment:
Should we use `unknown` instead of `any` in this file?

Reply:
Are you allowed to do comparison operations on `unknown`?

Reply:
Another option, probably the cleanest type-wise, is to make the function generic:
```ts
function defaultComp< A, B >( a: A, b: B ) {
  return a === b;
}
```

Reply:
Oops, I totally missed that feedback. I'll make sure to follow up with a PR!